k9mail/src/main/java/com/fsck/k9/controller/SnoozeController.java

```diff
@@ -0,0 +1,115 @@
+package com.fsck.k9.controller;
+
+import android.app.Activity;
+import android.app.AlarmManager;
+import android.app.Fragment;
+import android.app.Notification;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.content.res.Resources;
+import android.text.format.DateUtils;
+import android.widget.Toast;
+
+import com.fsck.k9.Account;
+import com.fsck.k9.R;
+import com.fsck.k9.activity.ChooseFolder;
+import com.fsck.k9.activity.ChooseSnooze;
+import com.fsck.k9.activity.MessageReference;
+import com.fsck.k9.notification.NotificationPublisher;
+
+/**
+ * Manages snoozing messages until later.
+ *
+ * Created by odbol on 12/10/16.
```

Comment:
Oops sorry my IDE adds this automatically! I totally agree though. At my work they make us write the author for each file and I think it's silly.